javascript/src/components/streamrules/CollapsableStreamRuleList.jsx

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var React = require('react/addons');
+var StreamRuleList = require('./StreamRuleList');
+var CollapsableMixin = require('react-bootstrap').CollapsableMixin;
+var Alert = require('react-bootstrap').Alert;
+
+var CollapsableStreamRuleList = React.createClass({
+    mixins: [CollapsableMixin],
+    getCollapsableDOMNode: function(){
+        return this.refs.well.getDOMNode();
+    },
+
+    getCollapsableDimensionValue: function(){
+        return this.refs.well.getDOMNode().scrollHeight;
```

Comment:
Please replace `.getDOMNode()` with `React.findDOMNode()`, as that's the recommended way of doing it. Don't forget the one in https://github.com/Graylog2/graylog2-web-interface/pull/1264/files#diff-16ded5d91aab616f09f72eb262b03346R11